### PR TITLE
Lighting fix 35

### PR DESCRIPTION
**Pull request by reosfire**
Some light compute improvements + fix section `y` light send threshold.

### CODE REVIEW

**Review by txfsjyqm — APPROVED** [review]
Works a bit better for me than version 34 from @iam4722202468 

At least I can't find any issues with broken skylight now.

File: src/main/java/net/minestom/server/instance/heightmap/HeightMapContainer.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package net.minestom.server.instance.heightmap;
+
+import net.minestom.server.instance.block.Block;
+import org.jglrxavpok.hephaistos.nbt.NBTCompound;
+
+public interface HeightMapContainer {
```
Comment (mworzala) [inline src/main/java/net/minestom/server/instance/heightmap/HeightMapContainer.java:6]: should be sealed permitting only the implementation

File: src/main/java/net/minestom/server/instance/heightmap/HeightMapContainerImpl.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package net.minestom.server.instance.heightmap;
+
+import net.minestom.server.instance.Chunk;
+import net.minestom.server.instance.block.Block;
+import org.jglrxavpok.hephaistos.nbt.NBT;
+import org.jglrxavpok.hephaistos.nbt.NBTCompound;
+
+import java.util.Map;
+
+public class HeightMapContainerImpl implements HeightMapContainer {
```
Comment (mworzala) [inline src/main/java/net/minestom/server/instance/heightmap/HeightMapContainerImpl.java:10]: should be package private

File: src/main/java/net/minestom/server/instance/heightmap/HeightMapContainer.java (hunk 1 above)
Comment (mworzala) [reply src/main/java/net/minestom/server/instance/heightmap/HeightMapContainer.java:6]: also mark apistatus internal

**Pull request merged**